cmd/podman/varlink/io.podman.varlink

```diff
@@ -434,8 +434,7 @@ method ListContainerProcesses(name: string, opts: []string) -> (container: []str
 method GetContainerLogs(name: string) -> (container: []string)
 
 # ListContainerChanges takes a name or ID of a container and returns changes between the container and
-# its base image. It returns a struct of changed, deleted, and added path names. If the
-# container cannot be found, a [ContainerNotFound](#ContainerNotFound) error will be returned.
```

Comment:
If ContainerNotFound is not returned, what behavior should I expect?

Reply:
youll get the standard error

Reply:
this was a case where the code was right and the API docs were wrong.